src/containers/org/DataSetMetaContainer.js

```diff
@@ -106,6 +113,10 @@ const DataSetMetaContainer = ({
 
   useEffect(() => {
     dispatch(getOwnerStatus(dataSetId));
+    dispatch(getCurrentDataSetAuthorizations({
+      acl: [dataSetId],
```

Comment:
This should technically be `aclKey`, but since the name implies a single data set, you could also change it to just `dataSetId`.